src/components/CourseTable/index.jsx

```diff
@@ -178,6 +171,13 @@ class CourseTable extends React.Component {
               placeholder="Search"
             />
           </div>
+          <div className="col-2 float-right">
```

Comment:
There is a weird empty space between the search bar and the new course button in this layout. Please fix that.

Reply:
fixed